src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -38,8 +38,11 @@ def handle_unverified_request
        when "branch?" then "branch"
        else exception.query
     end
-    message = "Sorry, you are not authorized to " +
-              (exception.message ? "#{pundit_action} this #{exception.record.class}." : "perform this action.")
+    if pundit_action && exception.record
+      message = "Sorry, you are not authorized to #{pundit_action} this #{exception.record.class}."
```

Comment:
NoMethodError was obviously an example;-)

Didn't see that we are just extracting the object's class from the exception.